packages/lib-user/src/components/shared/Select/Select.spec.js

```diff
@@ -0,0 +1,34 @@
+import { render, screen } from '@testing-library/react'
+import userEvent from '@testing-library/user-event'
+import { composeStory } from '@storybook/react'
+
+import Meta, { DateRanges } from './Select.stories'
+
+describe('components > shared > Select', function() {
+  const user = userEvent.setup()
+  const DefaultStory = composeStory(DateRanges, Meta)
+
+  it('should show the selected option', function() {
+    render(<DefaultStory />)
+
+    expect(screen.getByRole('textbox', { value: 'LAST 7 DAYS' })).to.be.ok()
+  })
+
+  it('should show the options', async function() {
+    render(<DefaultStory />)
+
+    const inputEl = screen.getByRole('textbox', { value: 'LAST 7 DAYS' })
+    await user.click(inputEl)
+
+    const options = screen.getAllByRole('option')
+
+    expect(options.length).to.equal(7)
+    expect(screen.getByText('LAST 7 DAYS')).to.be.ok()
```

Comment:
I'd throw the options into a shared file for the stories and specs to share for confirmation instead of having it hardcoded twice with an unknown dependency between them.